main.py

```diff
@@ -727,33 +730,78 @@ async def get_result(rid: str, result_file_type: ResultFileType) -> FileResponse
     return FileResponse(full_result_path)
 
 
+@app.websocket("/ttl/status/modification/")
+async def get_ttl_status_modification(websocket: WebSocket):
+    """Sends the modifications of TTL status whenever it is modified.
+    
+    After accepted, it receives the target TTL list.
+    Then, it sends the current TTL status immediately.
+    Finally, it sends the modifications of TTL status everty time it is modified.
+
+    See Modifications in the variables section of MonInj for modifications structure.
+
+    Args:
+        websocket: The web socket object.
+    """
+    await websocket.accept()
+    try:
+        devices = await websocket.receive_json()
+        latest, status = ttl_manager.current_status(devices)
+        await websocket.send_json(status)
+        while True:
+            latest, modifications = ttl_manager.modifications_since(devices, latest)
+            if not sum(len(m) for m in modifications.values()):  # no modification
```

Comment:
How about `if not any(modifications.values())`?

Reply:
Wow, I didn't come up with this idea! Thank you👍